src/main/scala/mesosphere/marathon/metrics/Metrics.scala

```diff
@@ -174,16 +175,15 @@ object Metrics {
   def snapshot(): TickMetricSnapshot = metrics
 
   // Starts collecting snapshots.
-  def start(actorRefFactory: ActorRefFactory): Done = {
+  def start(actorRefFactory: ActorRefFactory, averagingWindow: Duration): Done = {
```

Comment:
Hm, currently we pass conf objects. This makes it easier to add other metrics later on or fall back to Typesafe Conf. It seems there is a `MetricsReporterConf`. Why not pass that one?

Reply:
Yup, that works. Good idea 👍